apps/els_lsp/src/els_group_leader_server.erl

```diff
@@ -38,7 +38,6 @@
 %%==============================================================================
 %% Macro Definitions
 %%==============================================================================
--define(SERVER, ?MODULE).
```

Comment:
I think we should update the code to use ?SERVER.

Reply:
Fair point, fixed.